lib/resty/radixtree.lua

```diff
@@ -676,41 +685,31 @@ local function fetch_pat(path)
 end
 
 
-local function compare_param(req_path, route, opts)
+local function match_route_params(req_path, route, opts)
     if not opts.matched and not route.param then
-        return true
+        return true, nil, nil
```

Comment:
the `nil` is the default value, we do not need it here

we can revert this line

Reply:
@membphis remove  the trailing `nil`s in https://github.com/api7/lua-resty-radixtree/pull/142/commits/c32baae591dc9fa100d832b00d9960e8621ee2c1